server/game/cards/02.6-MotE/AlongTheRiverOfGold.js

```diff
@@ -3,12 +3,11 @@ const ProvinceCard = require('../../provincecard.js');
 class AlongTheRiverOfGold extends ProvinceCard {
     setupCardAbilities() {
         this.action({
-            title: 'During water Province chose a participating caracter, switch character base skils until turn',
-            condition: () => this.game.currentConflict.conflictProvince.getElement() === 'water',
-            target:{
+            title: 'During water Province chose a participating caracter, switch character base skils until end of turn',
+            condition: () => this.game.currentConflict.conflictProvince.getElement() === 'water' && this.game.currentConflict(),
```

Comment:
`this.game.currentConflict` isn't a function, it's either null or an object. You can just do `this.game.currentConflict`. Also, best to put this before `this.game.currentConflict.conflictProvince.getElement() === 'water'`, as that condition expects `this.game.currentConflict` to be not null.